com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -105,11 +105,13 @@ The Fabric Shader allows you to render realistic fabric Materials in HDRP. You c
 
 The AxF Shader supports the [X-Rite AxF](https://www.xrite.com/axf) measured Material format. The AxF importer, available in [Unity Enterprise for Product Lifecylce](https://unity.com/products/unity-enterprise-product-lifecycle), automatically populates an AxF Material when it imports AxF Assets. For more information, including a full list of Shader properties, see the [AxF Shader documentation](AxF-Shader).
 
-### Decal Shader
+### Decals
 
 ![](Images/HDRPFeatures-DecalShader.png)
 
-The Decal Shader allows you to place decals on surfaces in your Scene. To apply a decal to a surface, you can either use the [Decal Projector component](Decal-Projector.md) to project the decal onto the surface, or assign the decal directly to a Mesh and then place the Mesh on the surface. For more information, including the full list of Shader properties, see the [Decal Shader documentation](Decal-Shader.md).
+The Decal allows you to place decals on surfaces in your Scene. To apply a decal to a surface, you can either use the [Decal Projector component](Decal-Projector.md) to project the decal onto the surface, or assign the decal shader directly to a Mesh and then place the Mesh on the surface. For more information, including the full list of Shader properties, see the [Decal documentation](Decal.md).
```

Comment:
HDRP allows you to apply decals to surfaces in your Scene. To apply a decal to a surface, you can either use the [Decal Projector component](Decal-Projector.md) to project the decal onto the surface, or assign the decal shader directly to a Mesh and then place the Mesh on the surface. For more information, see the [Decal documentation](Decal.md).